### PR TITLE
[test] Add mem-per-cpu resources to ALLOC_SPEED check for Tiger

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix UES-567

### CODE REVIEW

**Comment by jgphpc** [comment]
```
[     FAIL ] AllocSpeedTest_no on dom:mc using PrgEnv-gnu
[     FAIL ] AllocSpeedTest_no on dom:gpu using PrgEnv-gnu

  * System partition: dom:gpu
  * Environment: PrgEnv-gnu
  * Stage directory: /scratch/snx3000tds/jenscscs/reframe-ci-dom-74fca3d-2556/stage/dom/gpu/PrgEnv-gnu/AllocSpeedTest_no
  * Job type: batch job (id=806389)
  * Maintainers: ['AK', 'VH']
  * Failing phase: performance
  * Reason: performance error: failed to meet reference: time=1.22354, expected 1.0 (l=-inf, u=1.1)
------------------------------------------------------------------------------
FAILURE INFO for AllocSpeedTest_no 
  * System partition: dom:mc
  * Environment: PrgEnv-gnu
  * Stage directory: /scratch/snx3000tds/jenscscs/reframe-ci-dom-74fca3d-2556/stage/dom/mc/PrgEnv-gnu/AllocSpeedTest_no
  * Job type: batch job (id=806390)
  * Maintainers: ['AK', 'VH']
  * Failing phase: performance
  * Reason: performance error: failed to meet reference: time=1.41507, expected 1.2 (l=-inf, u=1.32)
```

**Comment by jgphpc** [comment]
@victorusu is the reference too strict on dom ?

File: cscs-checks/microbenchmarks/alloc_speed/alloc_speed.py [hunk 1]
```diff
@@ -16,9 +16,16 @@ def __init__(self, hugepages):
...
-            if self.current_system.name in {'dom', 'daint'}:
+            if self.current_system.name in {'dom', 'daint', 'tiger'}:
                 self.modules = ['craype-hugepages%s' % hugepages]
 
+        if self.current_system.name in {'tiger'}:
+            self.extra_resources = {
+                'mem-per-cpu': {
+                    'mem_per_cpu': 0
+                }
+            }
```
Comment (vkarak) [inline cscs-checks/microbenchmarks/alloc_speed/alloc_speed.py]: @jgphpc I just noticed that I have read your question regarding this on Slack in the opposite way. You may also set launcher options directly if you wish. I've no strong preference.

Comment (jgphpc) [reply cscs-checks/microbenchmarks/alloc_speed/alloc_speed.py]: actually, `extra_resources` is the best way

**Comment by jgphpc** [comment]
@victorusu opened [issue992](https://github.com/eth-cscs/reframe/issues/992) about dom. This PR can be closed.

**Pull request merged**